### ISSUE TITLE
JWT Auth not working in default AddSwaggerDoc method

### ISSUE THREAD

**Issue by krjaction50** [dj-nitehawk/FastEndpoints#156]
AddSwaggerDoc() is by default supposed to add JWT auth support.  It seems the name "JWTBearerAuth" is the issue in EnableJWTBearerAuth is the issue.  When set up manually like this with "Bearer" as the name it works.

//builder.Services.AddSwaggerDoc(); //add this
builder.Services.AddSwaggerDoc(s =>
{
    s.AddAuth("Bearer", new()
    {
        Type = OpenApiSecuritySchemeType.Http,
        Scheme = "Bearer",
        BearerFormat = "JWT",
    });
});

**Comment by dj-nitehawk** [comment]
pause the video and double check each configuration line with your own startup code.
it works as intended.

**Comment by krjaction50** [comment]
I found the issue.  I used code from your sample in the Security documentation.

public override void Configure()
{
    Get("/account/profile");
    AuthSchems(JwtBearerDefaults.AuthenticationScheme);
}

The constant JwtBearerDefaults.AuthenticationScheme is equal to "Bearer" and that caused a mismatch from "JWTBearerAuth" which AddSwaggerDocs() uses.

So there's the mismatch.  If I change to using AddSwaggerDocs() and AuthSchems("JWTBearerAuth"); the padlocks are there.

FastEndpoints is saving us a lot of time on this new API we are creating.  Very nice component.
